packages/overlay/src/overlay-stack.ts

```diff
@@ -46,39 +46,24 @@ export class OverlayStack {
         return this.overlays.slice(-1)[0];
     }
 
-    public dispose(): void {
-        this.removeEventListeners();
-    }
-
     private addEventListeners(): void {
-        if (!this.document) return;
         this.document.addEventListener('click', this.handleMouseCapture, true);
         this.document.addEventListener('click', this.handleMouse);
         this.document.addEventListener('keyup', this.handleKeyUp);
         window.addEventListener('resize', this.handleResize);
     }
 
-    private removeEventListeners(): void {
```

Comment:
There's no longer a custom element with `disconnectedCallback` methods to connect this functionality to...